internal/pubsub/rabbitpubsub/rabbit.go

```diff
@@ -228,18 +211,12 @@ type subscription struct {
 
 var nextConsumer int64 // atomic
 
-func newSubscription(conn *amqp.Connection, name string) (*subscription, error) {
-	s := &subscription{
+func newSubscription(conn *amqp.Connection, name string) *subscription {
+	return &subscription{
 		conn:     conn,
 		queue:    name,
 		consumer: fmt.Sprintf("c%d", atomic.AddInt64(&nextConsumer, 1)),
 	}
-	s.mu.Lock()
-	defer s.mu.Unlock()
-	if err := s.establishChannel(); err != nil {
```

Comment:
Doesn't this call have to be done somewhere else now? I don't see it.

Reply:
It's in ReceiveBatch and SendAcks.

Reply:
Ah, right.